src/styles/LargeCard.scss

```diff
@@ -0,0 +1,19 @@
+@import "./Variables.scss";
+
+.lcm-photo {
+  padding-top: 3%;
+}
+
+.lcm-info {
+  width: 40%;
+}
+
+.card-lrg-bkgrnd {
```

Comment:
I would suggest changing the .card-lrg-bkgrnd to .card-lrg and then creating nested styles for the info area and the photo (see my two comments above). That way, all large cards will have the same styling